cldfbench_segbo.py

```diff
@@ -2,44 +2,112 @@
 import subprocess
 
 from cldfbench import Dataset as BaseDataset
+from cldfbench import CLDFSpec
 
 
 class Dataset(BaseDataset):
     dir = pathlib.Path(__file__).parent
     id = "segbo"
+    valueTableProperties = ['OnlyInLoanwords', 'Result', 'NewDistinction', 'PhonemeComments', 'Verified']
+    languageTableProperties = ['family_id', 'parent_id', 'bookkeeping', 'level', 'status', 'description', 'markup_description', 'child_family_count', 'child_language_count', 'child_dialect_count', 'country_ids']
 
     def cldf_specs(self):  # A dataset must declare all CLDF sets it creates.
-        from cldfbench import CLDFSpec
         return CLDFSpec(dir=self.cldf_dir, module='StructureDataset')
 
     def cmd_download(self, args):
         subprocess.check_call(
             'git -C {} submodule update --remote'.format(self.dir.resolve()), shell=True)
 
     def create_schema(self, ds):
+        # values.csv
+        ds.remove_columns('ValueTable', 'Code_ID', 'Comment')
+        ds.add_columns(
+            'ValueTable',
+            {
+                "dc:extent": "singlevalued",
+                "datatype": "string",
+                # FIXME: ValueError: Duplicate column property: http://cldf.clld.org/v1.0/terms.rdf#languageReference
+                # "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#languageReference",
+                "required": True,
+                "name": "Source_Language_ID",
```

Comment:
I think I'd make this multivalued, splitting the content on comma and filtering out `unknown`. Then you could also assign `propertyUrl` https://cldf.clld.org/v1.0/terms.rdf#glottocode

Reply:
I was trying to make it a languageReference, a https://cldf.clld.org/v1.0/terms.rdf#glottocode should also work, thanks!

Reply:
Yes, `languageReference` won't work for two reasons:
- There is already a `languageReference` in the table (and somewhat unfortunately, propertyUrls in the table need to be unique).
- If the dataset has a LanguageTable, languageReferences would need to be present in this table.

Anyway, I think `glottocode` is the right choice here.